be/src/formats/parquet/file_reader.cpp

```diff
@@ -503,7 +503,16 @@ Status FileReader::_init_group_readers() {
             auto row_group_reader =
                     std::make_shared<GroupReader>(_group_reader_param, i, _need_skip_rowids, row_group_first_row);
             _row_group_readers.emplace_back(row_group_reader);
-            _total_row_count += _file_metadata->t_metadata().row_groups[i].num_rows;
+            int64_t num_rows = _file_metadata->t_metadata().row_groups[i].num_rows;
+            // for iceberg v2 pos delete
```

Comment:
what's wrong if we don't handle delete?  wrong total_row_count?

Reply:
yes wrong total_row_count